photon-ml/src/integTest/scala/com/linkedin/photon/ml/cli/game/training/DriverTest.scala

```diff
@@ -560,7 +391,7 @@ class DriverTest extends SparkTestUtils with TestTemplateWithTmpDir {
     * @param args the command-line arguments
     */
   def runDriver(args: Array[String]): Driver = {
-    val params = Params.parseFromCommandLine(args)
+    val params = GameParams.parseFromCommandLine(args)
```

Comment:
Holding back on changes to Params, because this is going through a big overhaul anyway with Alex's work.